src/server/middleware/removePreviousStoryFile.js

```diff
@@ -0,0 +1,49 @@
+/*
+For copyright information, see the AUTHORS.md file in the docs directory of this distribution and at
+https://github.com/fluid-project/sjrk-story-telling/blob/master/docs/AUTHORS.md
+
+Licensed under the New BSD license. You may not use this file except in compliance with this licence.
+You may obtain a copy of the BSD License at
+https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENSE.txt
+*/
+
+"use strict";
+
+var fluid = require("infusion");
+var path = require("path");
+var fs = require("fs");
+require("kettle");
+
+var sjrk = fluid.registerNamespace("sjrk");
+
+// Middleware to remove the previous file associated with a story block
+fluid.defaults("sjrk.storyTelling.server.middleware.removePreviousStoryFile", {
```

Comment:
Consider renaming, as it appears that this can remove any file, not just the previous one.